tests/data/power_flow/asym-line/status-from0-to1/test.dss

```diff
@@ -0,0 +1,29 @@
+Clear
+
+Set DefaultBaseFrequency=50
+
+! Swing or Source Bar
+New circuit.test_1 phases=3 pu=1.0 basekv=10 bus1=Node1.1.2.3.4 MVAsc3=1000000000 MVAsc1=1000000000.0 x1r1=0.166666 x0r0=0.166666
+
+! LineCodes
+New LineCode.LineCode1 nphases=4 C1=0.54 C0=0.32 Normamps=115.0 Units=km
+~ Rmatrix = [0.6904 |0.0495 0.6904 |0.0492 0.0495 0.6904 | 0.0495 0.0492 0.0495 0.6904]
+~ Xmatrix = [0.8316 |0.7559 0.8316 |0.7339 0.7559 0.8316 | 0.7559 0.7339 0.7559 0.8316]
+~ Neutral=4 Kron=yes ! eliminate the neutral conductor (4)
```

Comment:
Since we are bumping into the issue with neutrals and kron reduction, Lets change this case to Rmatrix Xmatrix Cmatrix with the switches. No neutrals or parameters. There shouldn't be any difference in modelling with one side active lines with / without kron reduced neutral. 

We can atleast check these status=0,1 and 1,0 cases off of our list.
```suggestion
~ Rmatrix = [0.6904 |0.0495 0.6904 |0.0492 0.0495 0.6904 | 0.0495 0.0492 0.0495 0.6904]
~ Xmatrix = [0.8316 |0.7559 0.8316 |0.7339 0.7559 0.8316 | 0.7559 0.7339 0.7559 0.8316]
~ Neutral=4 Kron=yes ! eliminate the neutral conductor (4)
```

Reply:
why not both with neutral and without?

Reply:
Both situations end up at same point logic wise. We get a 3x3 y ff/ft/tf/tt and the distribution is same for different status situations.